mmv1/products/datacatalog/terraform.yaml

```diff
@@ -98,7 +98,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           regex: '^[a-z_][a-z0-9_]{0,63}$'
       fields: !ruby/object:Overrides::Terraform::PropertyOverride
         description: |
-          Set of tag template field IDs and the settings for the field. This set is an exhaustive list of the allowed fields. This set must contain at least one field and at most 500 fields.
+          Set of tag template field IDs and the settings for the field. This set is an exhaustive list of the allowed fields. This set must contain at least one field and at most 500 fields. The change of field_id will be resulting in re-creating of field.
```

Comment:
Do we also want to mention that change of field primitiveType will also result in re-creating of the field?

Reply:
Sure, we can add as it is an internal behavior for the primitiveType.